src/kirby/components/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,27 @@
+<button
+  kirby-button
+  [attentionLevel]="isOpen ? attentionLevelOpen : attentionLevel"
+  tabindex="-1"
+  (click)="onToggle()"
+  (mousedown)="onButtonMouseEvent($event)"
+>
+  {{ selectedText }}
+  <kirby-icon [name]="isOpen ? 'arrow-up' : 'arrow-down'"></kirby-icon>
+</button>
+<kirby-card>
```

Comment:
The list of options should have a `listbox` role and every option should have an `option` role (see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role)

Reply:
The list already has the `listbox` role (see: https://github.com/kirbydesign/designsystem/pull/899/files#diff-a963934da666cc58e9394c82c15b0d91R43)
Good point regarding the `optionn` role, I'll add that 👍

Reply:
Added 👍